cloudferrylib/os/compute/server_groups.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2014 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+import pprint
+
+from novaclient import exceptions as nova_exc
+
+from cloudferrylib.base.resource import Resource
+from cloudferrylib.utils import utils as utl
+
+
+LOG = utl.get_log(__name__)
+
+
+class Handler(Resource):
```

Comment:
The class name seems quite a bit too generic, given the number of SQL requests done in class methods, I guess it should be named differently and placed in a different module